src/util/createdb.cpp

```diff
@@ -19,6 +19,28 @@ int createdb(int argc, const char **argv, const Command& command) {
     std::vector<std::string> filenames(par.filenames);
     std::string dataFile = filenames.back();
     filenames.pop_back();
+    if (Util::endsWith(".txt", filenames[0])) {
```

Comment:
Please use `.tsv`.

Also please adjust MMseqsBase.cpp to explain that users can pass in a tsv

Reply:
In the createdb module only, or in all search/cluster modules as well?